governance/polls/Proposal - Change the Stability Fee Structure - May 18 2020.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Change the Stability Fee Structure - May 18 2020
+summary: Signal your support to change the structure of the weekly Stability Fee votes.
+discussion_link: https://forum.makerdao.com/t/2380
+poll_rules: The voter may select to vote for one or more of the poll options or they may elect to abstain from the poll entirely
```

Comment:
```suggestion
vote_type: Ranked Choice IRV
```
this is the only change you need to make to mark this poll as ranked choice :) (see [this PR](https://github.com/makerdao/community/pull/540/files) for another example of poll designed to be RC)